etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/ruleengine/transformation/custom/CopyOrcOrderProviderToObrOrderProvider.java

```diff
@@ -28,16 +28,11 @@ public void transform(HealthData<?> resource, Map<String, Object> args) {
             return;
         }
 
-        // Extract or create the OBR-16 extension from the ServiceRequest
+        // Extract or create the OBR extension from the ServiceRequest
         Extension obrExtension =
                 HapiHelper.ensureExtensionExists(serviceRequest, HapiHelper.EXTENSION_OBR_URL);
 
-        // Extract or create the OBR-16 data type extension
-        Extension obr16Extension =
```

Comment:
What would happen if the extension doesn't exist? Is this not needed?

Reply:
This was refactored into `HapiHelper.setOBR16WithPractitioner` so we can be sure we're not creating an extension if there's nothing to put in it.